pyproject.toml

```diff
@@ -42,6 +42,9 @@ classifiers = [
 "Source code" = "https://github.com/SainsburyWellcomeCentre/WAZP"
 "User support" = "https://github.com/SainsburyWellcomeCentre/WAZP/issues"
 
+[project.scripts]
+startwazp = "wazp.app:run"
```

Comment:
I didn't like `wazp` because it's not clear that you're actually _starting_ something. WAZP isn't really a tool. V. v. soft preference though.

Reply:
I may be missing something obvious but I get an `ImportError` error after doing this:
1- `pip install .` from the WAZP top directory
2- run `startwazp`
3- `ImportError: cannot import name 'run' from 'wazp.app'`
Do we need a `run()` function in `app.py`?